zerver/tests/tests.py

```diff
@@ -2132,7 +2132,7 @@ def test_open_realm_logic(self):
         mit_realm.save()
         self.assertTrue(completely_open(mit_realm.domain))
         self.assertEquals(get_unique_open_realm(), None)
-        with self.settings(SYSTEM_ONLY_REALMS={"zulip.com"}):
+        with self.settings(SYSTEM_ONLY_REALMS={"zulip.com", "analytics.test"}):
```

Comment:
hm, putting
```
    def tearDown(self):
        # type: () -> None
        Realm.objects.filter(domain='analytics.test').delete()
```
in AnalyticsTestCase didn't help. Any other ideas? (Putting `print(Realm.objects.all())`'s does verify that the tearDown is being run and the analytics.test realm is being deleted after each analytics test case.)